extensions/amp-story/1.0/animation-presets.js

```diff
@@ -232,17 +232,17 @@ export const getPresetDef = (name, options) => {
       };
     case 'whoosh-in-left':
       return {
-        duration: 500,
-        easing: 'ease-out',
+        duration: 400,
+        easing: 'cubic-bezier(0.0, 0.0, 0.2, 1)',
         keyframes(dimensions) {
           const offsetX = -(dimensions.targetX + dimensions.targetWidth);
           return whooshIn(offsetX, 0, 0, 0);
         },
       };
     case 'whoosh-in-right':
       return {
-        duration: 500,
-        easing: 'ease-out',
+        duration: 400,
+        easing: 'cubic-bezier(0.0, 0.0, 0.2, 1)',
```

Comment:
Should we also add these to pan-up/down/.. & zoom-in/out ? I heard there was a request to do this. Up to you!